Sources/NIOHTTP1/HTTPTypes.swift

```diff
@@ -191,6 +191,29 @@ extension HTTPRequestHead {
     }
 }
 
+extension HTTPResponseHead {
+    /// Whether this HTTP response is a keep-alive request: that is, whether the
+    /// connection should remain open after the request is complete.
+    public var isKeepAlive: Bool {
```

Comment:
I think we should implement it kind of the same way as here:

https://github.com/apple/swift-nio/pull/402/files#diff-9361bcb5608a311475f3a33eeadc3a3eR300

Reply:
@normanmaurer ok, will do that tomorrow. Also the 'wrong' impl first?

Reply:
@weissi let me just merge my pr and then you can reuse the `headers.isKeepAlive`